### ISSUE TITLE
blocking issue on motor runs on ev3dev2 (maybe ev3dev too)

### ISSUE THREAD

**Issue by omartin2010** [ev3dev/ev3dev-lang-python#558]
- **ev3dev version:** 4.14.71-ev3dev-2.3.0-rpi2
- **ev3dev-lang-python version:** 
```
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                        Version            Architecture       Description
+++-===========================-==================-==================-===========================================================
ii  python3-ev3dev2             2.0.0~beta2        all                Python language bindings for ev3dev
```

So running m.on_for_degrees(-30, 90, block=True) works on the Lego brick (returning control when position is reached) and the same command on the Dexter brick doesn't return control. The code for repro is : 

```
from ev3dev2.motor import LargeMotor, OUTPUT_A, SpeedPercent
m = LargeMotor(OUTPUT_A)
m.on_for_degrees(SpeedPercent(10), 180, block = True)

```
This will be stuck forever on the dexter (polling issue?) but not on the lego brick. We can repro on 2 different Dexter devices with EV3 and NXT.

**Comment by WasabiFan** [comment]
@dwalton76 Do you have a BrickPi you can test this on?

**Comment by omartin2010** [comment]
adding @lynn-orell

**Comment by dwalton76** [comment]
This should get fixed via #548

**Comment by WasabiFan** [comment]
Ah, I had forgotten about that one; thanks for staying on top of it! I just edited the PR description to link back with `fixes`.
